app/controllers/ctc/questions/filed_prior_tax_year_controller.rb

```diff
@@ -1,16 +1,12 @@
 module Ctc
   module Questions
-    class Filed2019Controller < QuestionsController
+    class FiledPriorTaxYearController < QuestionsController
       include Ctc::ResetToStartIfIntakeNotPersistedConcern
 
       layout "intake"
 
       private
 
-      def method_name
```

Comment:
Wasn't able to find any concrete calls (but suspecting its getting called in more sneaky ways) but just wanted ton double check whether or not we should just replace it with a new method name instead of removing it entirely?

Reply:
Check out the parent controller to this one. The default behavior is to use the method name that matches the controller name without controller. Here, `filed_prior_tax_year` is the default name that gets created.